src/pages/patterns/buttons.mdx

```diff
@@ -0,0 +1,387 @@
+It can be frustrating if a user doesn’t know what mistake they made and how to proceed.
+
+<BestPractice>
+  <Do
+    imageIsFreeForm
+    imageSource={props.data.buttonsDisabledDo.childFile.childImageSharp.gatsbyImageData}
+  >
+    <ul>
+      <li>Always keep buttons interactive</li>
+      <li>Add contextual, in-line error notifications that set people up for success</li>
+    </ul>
+  </Do>
+  <Dont
+    imageIsFreeForm
+    imageSource={props.data.buttonsDisabledDont.childFile.childImageSharp.gatsbyImageData}
+  >
+    <ul>
+      <li>Do not disable a button with no clear direction on how to move forward</li>
+    </ul>
+  </Dont>
+</BestPractice>
+
+## Flows
+
+### Buttons vs Anchor (links)
+
+![Buttons vs Anchor links](figma:patterns-buttons-button-vs-link.png)
+
+1. [Buttons](/components/button) trigger an action on a page or view
+2. [Anchor](/components/anchor) (links) help users navigate from one location to another
+
+## Content guidelines
+
+The button call-to-action (CTA) should guide the experience, not just finish a page.
+Typically, the CTA repeats the page heading when possible (e.g. “Add agent”).
+The phrase repetition confirms the action that will take place.
+
+<BestPractice>
+  <Do
+    imageIsFreeForm
+    imageSource={props.data.buttonsContentDo.childFile.childImageSharp.gatsbyImageData}
+  >
+    <ul>
+      <li>Write labels as verbs</li>
+      <li>Communicate a single action per button</li>
+      <li>Use sentence case</li>
+      <li>Consider longer languages. Use 1-4 words and a maximum of 20 characters</li>
+    </ul>
+  </Do>
+  <Dont
+    imageIsFreeForm
+    imageSource={props.data.buttonsContentDont.childFile.childImageSharp.gatsbyImageData}
+  >
+    <ul>
+      <li>Avoid CTAs that lack context like "Yes," "No," or "Maybe"</li>
+      <li>Avoid calling out more than one action</li>
+      <li>Avoid ALL CAPS</li>
+      <li>Avoid articles, pronouns, and punctuation</li>
+    </ul>
+  </Dont>
+</BestPractice>
+
+For more help with choosing the right verb, visit our [actionable language guidelines](/content/actionable-language).
+
+## Localization & internationalization
+
+### RTL (right-to-left)
+
+Actions need to accommodate different languages (making them wider or shorter) and both left-to-right
+and right-to-left direction. Keep in mind that the width of your actions will change and act accordingly.
+
+For RTL languages, the layout of the button is mirrored.
+[Split buttons](/components/split-button) will automatically switch places for dropdown and label,
+whilst media elements such as icons will be placed on the right side of the text.
+
+<ImageFigure
+  caption="Arabic RTL example"
+  imgAlt="Arabic RTL example"
+  imageSource={props.data.buttonsLocalizationArabic.childFile.childImageSharp.gatsbyImageData}
+/>
+
+## Accessibility
+
+### Keyboard support
+
+Users must be able activate a button by pressing `Enter` or `Space` while the button has focus.
```

Comment:
```suggestion
Users must be able to activate a button by pressing `Enter` or `Space` while the button has focus.
```

Reply:
@jzempel Thank you for the extra eagle eyes.